packages/synthetics-sdk-mocha/package.json

```diff
@@ -34,8 +34,8 @@
     "synthetics-sdk-mocha": "file:./"
   },
   "dependencies": {
-    "@google-cloud/synthetics-sdk-api": "^0.1.0",
-    "mocha": "^10.2.0",
-    "error-stack-parser": "^2.1.4"
+    "@google-cloud/synthetics-sdk-api": "google-cloud-synthetics-sdk-api-0.1.0.tgz",
```

Comment:
did u mean to change this from ^0.1.0

Reply:
yes that actually is there on purpose. npm install isnt working at the workspace level within workflows. to account for that I need to do things in the following order...
1) Release synthetics-sdk-api to npm
2) update the github package for synthetics-sdk-mocha to point to that
3) release synthetics-sdk-mocha to npm